src/header/header.scss

```diff
@@ -263,6 +375,20 @@
           box-shadow: 0 $iui-xxs 0 0 t(iui-color-foreground-primary);
         }
       }
+
+      .iui-split-menu:hover {
```

Comment:
You have above also rule for hover and svg... Not sure which should be when now, but seems like repeating...
![image](https://user-images.githubusercontent.com/81580355/127849248-67636cb8-c42d-46d2-acbf-9add7be7e65f.png)

Reply:
Styling was changed.